RadarSDK/RadarLocationManager.m

```diff
@@ -94,6 +94,58 @@ - (instancetype)init {
         _lowPowerLocationManager.distanceFilter = 3000;
         _lowPowerLocationManager.allowsBackgroundLocationUpdates = [RadarUtils locationBackgroundMode];
 
+        RadarFeatureSettings *featureSettings = [RadarSettings featureSettings];
+        if (featureSettings.useLocationMetadata) { 
+            [_locationManager startUpdatingHeading];
+
+            _motionActivityManager = [CMMotionActivityManager new];
+            [_motionActivityManager startActivityUpdatesToQueue:[NSOperationQueue mainQueue] withHandler:^(CMMotionActivity * _Nullable activity) {
+                if (activity) {
+                    [RadarState setLastMotionActivityData:@{
+                        @"stationary" : @(activity.stationary),
+                        @"walking" : @(activity.walking),
+                        @"running" : @(activity.running),
+                        @"driving" : @(activity.automotive),
+                        @"cycling" : @(activity.cycling),
+                        @"unknown" : @(activity.unknown),
+                        @"timestamp" : @([activity.startDate timeIntervalSince1970]),
+                        @"confidence" : @(activity.confidence)
+                    }];
+                }
+            }];
+        
+            _motionManager = [CMMotionManager new];
```

Comment:
We are creating an instance of CMMotionManager on the main thread, apparently the init of the manager is slow and we should do this on another thread.
https://stackoverflow.com/questions/55629406/ios-12-2-cmmotionmanager-is-blocking-the-main-thread